listenbrainz/webserver/static/js/src/follow/FollowerFollowingModal.tsx

```diff
@@ -50,13 +56,15 @@ export default class FollowerFollowingModal extends React.Component<
         <div className="text-center follower-following-pills">
           <div className="btn-group btn-group-justified" role="group">
             <Pill
+              style={{ margin: "0px 6px" }}
```

Comment:
This would be better done in the follow.less file where `.follower-following-pills` is defined

Reply:
Ah, I need to improve my relationship with CSS 😅

Reply:
It's a good skill to have :)

To be honest, for simple things its very accessible.